src/main/java/randoop/generation/InputSequenceSelector.java

```diff
@@ -0,0 +1,29 @@
+package randoop.generation;
+
+import java.util.Set;
+import randoop.sequence.ExecutableSequence;
+import randoop.sequence.Sequence;
+import randoop.util.SimpleList;
+
+/** Interface for selecting sequences as input for creating new sequences. */
+public interface InputSequenceSelector {
+  /**
+   * Choose a sequence used as input for creating a new sequence.
+   *
+   * @param candidates sequences to choose from
+   * @return the chosen sequence
+   */
+  Sequence selectInputSequence(SimpleList<Sequence> candidates);
+
+  /**
+   * Each {@link Sequence} within {@code inputSequences} is a subsequence of {@code eSeq}. At this
+   * point, the given {@link ExecutableSequence} has been executed and contains the execution time
+   * of the sequence as a whole. Make use of the execution time value that is stored within eSeq and
+   * associate it with each of the input sequences.
```

Comment:
I believe your interpretation is correct. Here I was making the simplifying assumption that an input sequence's execution time would be roughly similar to the execution of the complete sequence. I reasoned that since an input sequence's execution is measured and stored upon its first inclusion in a complete sequence that is run, the complete sequence would be relatively short and that this approximation would be close.
However, I do agree that this doesn't exactly match what is described in the GRT paper.
To address this, I have changed Orienteering to compute the execution time of each input sequence. Previously, I was just retrieving the exec_time variable from the eSeq that gets executed. Now, for each input sequence, I iterate through its statements and find the corresponding statement within the eSeq. Each statement in eSeq is also associated with an ExecutionOutcome which stores the execution time of the statement. The execution time of the input sequence is then the sum of the execution times of the statements.